index.html

```diff
@@ -364,6 +368,12 @@ <h1>bam2.iobio</h1>
         console.log(event.detail)
       });
 
+      document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Do you need to wait on DOMContentLoaded here? I think since the script is below the HTML and the web component initialization code it should work without.

Reply:
I think you're right. The script is at the bottom of the body. The component should be accessible.